source/backend/apimodels/Models/Concepts/Document/DocumentMap.cs

```diff
@@ -14,6 +15,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.DocumentType, src => src.DocumentType)
                 .Map(dest => dest.StatusTypeCode, src => src.DocumentStatusTypeCodeNavigation)
                 .Map(dest => dest.FileName, src => src.FileName)
+                .Map(dest => dest.DocumentQueueStatusTypeCode, src => src.PimsDocumentQueues.Count > 0 ? src.PimsDocumentQueues.FirstOrDefault().DocumentQueueStatusTypeCodeNavigation : null)
```

Comment:
This is an interesting one, as generally we try to pass concepts as-is without merging entity fields. Thoughts @FuriousLlama ?